src/lib/validators/createTrackedRepoValidator.ts

```diff
@@ -0,0 +1,5 @@
+import z from 'zod';
+
+export const createTrackedRepoValidator = z.object({
```

Comment:
Since we've nested the user and repo resources in the route already, the correct naming convention would be to  call these UserRepos to differentiate them from repos that have not been added by a user (as opposed to TrackedRepos).